.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,176 @@
+      placeholder: |
+        ```sh
+        [Sample Test] Test Suite
+        ────────────────────────────────────────────────────────
+        ⠋ Starting ChromeDriver on port 9515...
+        ⠙ Starting ChromeDriver on port 9515...
+          Request POST /session
+          {
+            desiredCapabilities: {
+              browserName: 'chrome',
+              'goog:chromeOptions': { w3c: true, args: [] },
+              name: 'Sample Test'
+            },
+            capabilities: {
+              alwaysMatch: {
+                browserName: 'chrome',
+                'goog:chromeOptions': { w3c: true, args: [] }
+              }
+            }
+        ⠸ Starting ChromeDriver on port 9515...
+        ```
+  - type: textarea
+    id: "configuration"
+    attributes:
+      label: "Nightwatch Configuration"
+      description: "Add your nightwatch.json or nightwatch.conf.js here; Make sure to leave out any sensitive details."
+      placeholder: |
+        //
+        // Refer to the online docs for more details:
+        // https://nightwatchjs.org/gettingstarted/configuration/
+        //
+        //  _   _  _         _      _                     _          _
+        // | \ | |(_)       | |    | |                   | |        | |
+        // |  \| | _   __ _ | |__  | |_ __      __  __ _ | |_   ___ | |__
+        // | . ` || | / _` || '_ \ | __|\ \ /\ / / / _` || __| / __|| '_ \
+        // | |\  || || (_| || | | || |_  \ V  V / | (_| || |_ | (__ | | | |
+        // \_| \_/|_| \__, ||_| |_| \__|  \_/\_/   \__,_| \__| \___||_| |_|
+        //             __/ |
+        //            |___/
+        //
+        module.exports = {
+          src_folders: [],
+          page_objects_path: [],
+          custom_commands_path: [],
+          custom_assertions_path: '',
+          plugins: [],
+          globals_path: '',
+          webdriver: {},
+          test_settings: {}
+        }
+    validations:
+      required: true
+  - type: "markdown"
+    attributes:
+      value: |
+        ## Your Environment
+        Include the relevant details related to your environment.
+  - type: "input"
+    id: "nightwatch-version"
+    attributes:
+      label: "Nightwatch Js Version"
+      description: "The version of Nightwatch Js you use."
+      placeholder: "2.1.7"
+    validations:
+      required: true
+  - type: "input"
+    id: "npm-version"
+    attributes:
+      label: "NPM Version"
+      description: "The version of NPM you use."
+      placeholder: "8.12.1"
+    validations:
+      required: true
+  - type: "input"
+    id: "node-version"
+    attributes:
+      label: "Node Version"
+      description: "The version of Node you use."
+      placeholder: "8.12.1"
+  - type: "input"
+    id: "yarn-version"
+    attributes:
+      label: "Yarn Version"
+      description: "The version of Yarn you use."
+      placeholder: "1.0.1"
+  - type: "input"
+    id: "browser"
+    attributes:
+      label: "Browser"
```

Comment:
this should be a list, with a version "All"

Reply:
So we don't care about the browser version. We just care about the browser rt? I was thinking on the same line, but previously we use to ask both version and browser, so I added here as an input instead of list

Reply:
Updated in latest commit

Reply:
You left out Edge. Browser version is also important though, so maybe leave it as a text field and add a placeholder like "Firefox 101.0.1"

Reply:
^ the problem with the Text field approach is people generally skip adding relevant information or only add browser name

Reply:
I am gonna update as you suggested, but my suggestion would be:
- Have Browser has an options
- Have another text field like Version

Reply:
Updated in latest commit